javafx-archetype-simple/src/main/resources/archetype-resources/pom.xml

```diff
@@ -6,24 +6,26 @@
     <version>${version}</version>
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <maven.compiler.source>11</maven.compiler.source>
-        <maven.compiler.target>11</maven.compiler.target>
+#if ("8" == ${javafx-version})
+            <maven.compiler.source>8</maven.compiler.source>
+            <maven.compiler.target>8</maven.compiler.target>
+#else
+            <maven.compiler.source>11</maven.compiler.source>
+            <maven.compiler.target>11</maven.compiler.target>
+#end
     </properties>
-    <dependencies>
-        <dependency>
-            <groupId>org.openjfx</groupId>
-            <artifactId>javafx-controls</artifactId>
-            <version>${javafx-version}</version>
-        </dependency>
-    </dependencies>
     <build>
         <plugins>
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>3.8.0</version>
                 <configuration>
+#if ("8" == ${javafx-version})
+                    <excludes>module-info.java</excludes>
```

Comment:
According to http://maven.apache.org/plugins/maven-compiler-plugin/examples/module-info.html, this should probably be:
```
<excludes>
    <exclude>module-info.java</exclude>
</excludes>
```